### PR TITLE
powerstate from phidget

### PR DESCRIPTION
**Pull request by benmaidel**
get powerstate informations from phidget board as long as the bms system is buggy

depends on:
- https://github.com/ipa320/cob_driver/pull/279

### CODE REVIEW

**Comment by floweisshardt** [comment]
as this is a cob4-1/msh specific and temporary fix we should NOT merge this. Should be solved within the msh repository

**Comment by benmaidel** [comment]
I do not agree. This is not a msh specific fix. This is a cob4 temporary fix! As soon as the bms is fixed we can switch back.

**Comment by floweisshardt** [comment]
as discussed with @ipa-bnm we'll do the following: 
- create two new packages in cob_driver named cob_phidgets_power_state and cob_phidgets_emergency_stop
- split cob_voltage_control into the two new packages
- keep cob_bms_driver clean from phidget related things

**Comment by floweisshardt** [comment]
CR: OK

**Comment by floweisshardt** [comment]
merge together with https://github.com/ipa320/cob_driver/pull/279

**Comment by floweisshardt** [comment]
@ipa-bnm: please add a dependency to cob_driver, see https://github.com/ipa320/cob_driver/blob/indigo_dev/.travis.rosinstall

File: cob_bringup/drivers/powerstate_phidget.launch [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<launch>
+
+	<node name="power_state_phidget" pkg="cob_phidgets_power_state" type="power_state_phidget.py" output="screen">
```
Comment (floweisshardt) [inline cob_bringup/drivers/powerstate_phidget.launch]: the package is called "cob_phidget_power_state" (withous "s")

File: cob_bringup/package.xml [hunk 2]
```diff
@@ -45,6 +45,7 @@
   <exec_depend>cob_obstacle_distance</exec_depend>
   <exec_depend>cob_obstacle_distance_moveit</exec_depend>
   <exec_depend>cob_phidgets</exec_depend>
+  <exec_depend>cob_phidgets_power_state</exec_depend>
```
Comment (floweisshardt) [inline cob_bringup/package.xml]: the package is called "cob_phidget_power_state" (withous "s")

File: cob_bringup/robots/cob4-1.xml [hunk 3]
```diff
@@ -81,9 +81,10 @@
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="component_name" value="light_base" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/bms.launch">
+		<!--include file="$(find cob_bringup)/drivers/bms.launch">
 			<arg name="robot" value="$(arg robot)" />
-		</include>
+		</include-->
+                <include file="$(find cob_bringup)/drivers/powerstate_phidget.launch"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-1.xml]: please use tabs instead of spaces

**Comment by floweisshardt** [comment]
CR: OK

**Pull request merged**